client/cmd/dexcctl/dexcctl_test.go

```diff
@@ -75,20 +75,20 @@ func TestConfigure(t *testing.T) {
 	}
 
 	// parse args
-	os.Args = []string{"", "-C.nofile", "arg1", "arg2"}
+	os.Args = []string{"", "-C.nofile", "arg1", "arg2", "-1"}
 	_, args, _, err := configure()
 	if err != nil {
 		t.Fatal(err)
 	}
-	if args[0] != "arg1" && args[1] != "arg2" {
+	if args[0] != "arg1" || args[1] != "arg2" || args[2] != "-1" {
 		t.Fatal("arguments not parsed correctly")
 	}
 
 	// bad flag
 	os.Args = []string{"", "-7"}
 	_, _, _, err = configure()
 	if err == nil {
-		t.Fatal("expected failure on bad flag")
+		t.Fatal("expected success on bad flag")
```

Comment:
`expected success` here doesn't add up with `if err == nil` check from above ? Not sure what's the intention, just caught my eye.